### PR TITLE
(fix) - restore debounceRendering after act

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Fixes: https://github.com/preactjs/preact/issues/1681

### CODE REVIEW

File: test-utils/src/index.js [hunk 1]
```diff
@@ -36,6 +37,7 @@ export function act(cb) {
 		rerender();
 	}
 
+	options.debounceRendering = previousDebounceRendering;
```
Comment (andrewiggins) [inline test-utils/src/index.js]: Should `act` call `teardown` since it calls `setupRerender()`? `teardown` not only restores `debounceRendering` but also flushes any updates called so that the next test begins with an empty update queue.

Comment (robertknight) [reply test-utils/src/index.js]: My thought is the same reading this. I think the setup and teardown processes should be symmetrical in `act`. Otherwise someone might add code to `setupRerender` in future and that wouldn't get undone at the end of `act`.

**Comment by JoviDeCroock** [comment]
@robertknight it should be fixed now
